app/templates/components/paper-item.hbs

```diff
@@ -1,11 +1,3 @@
-{{#if action}}
-  {{#paper-button class="md-no-style" noink=true onClick="buttonAction" skipProxy=true}}
-    <div class="md-list-item-inner">
-      {{yield}}
-    </div>
-  {{/paper-button}}
-{{else}}
-  <div class="md-no-style md-list-item-inner" {{action "buttonAction" on="click"}}>
-    {{yield}}
-  </div>
-{{/if}}
+<div class="md-no-style md-list-item-inner" {{action "buttonAction" on="click"}}>
```

Comment:
@miguelcobain seems that it is something different from just a notation. Changing this line breaks 2 tests. For some reason if we select the nested `paper-button` element with jquery and do a `click()` on it this `onClick` handler would run prior to the one of `paper-button` `click(event)` handler which should actually run first as event is directly targeted on it thus ignoring `paper-button` `bubbles` prop. Any ideas why this might happen?